Common/utils.ts

```diff
@@ -220,4 +221,187 @@ export function ObjectArrayToCsv(objects: { [key: string]: any }[], headers?: st
   ));
 
   return [dictionaryKeys.join(','), ...dictValuesAsCsv].join('\n');
+}
+
+export function combine(object1: PlayerStatModel, object2: PlayerStatModel): PlayerStatModel {
```

Comment:
Ik this function is duplicating code, but i will refactor when i do the next task

Reply:
Ok